pipelines/_ingredients/ApVerify.yaml

```diff
@@ -3,10 +3,10 @@
 
 description: Fully instrumented AP pipeline
 imports:
-  - location: $AP_PIPE_DIR/pipelines/ApPipe.yaml
-  - location: $AP_VERIFY_DIR/pipelines/MetricsRuntime.yaml
-  - location: $AP_VERIFY_DIR/pipelines/MetricsMisc.yaml
-  - location: $AP_VERIFY_DIR/pipelines/Conversions.yaml
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipe.yaml
+  - location: $AP_VERIFY_DIR/pipelines/_ingredients/MetricsRuntime.yaml
+  - location: $AP_VERIFY_DIR/pipelines/_ingredients/MetricsMisc.yaml
+  - location: $AP_VERIFY_DIR/pipelines/_ingredients/Conversions.yaml
```

Comment:
The "not meant to be run" status of the metrics pipelines is a bit messy. It's true that they're designed to be used as part of `ApVerify*.yaml`, mostly because they depend on external `parameter` definitions, but they're also (supposed to be) frequently updated as developers add new metrics for AP testing. I'm worried that this second usage will be harmed by hiding them in `_ingredients`.

Reply:
It is *not* intended that developers should shy away from making any commits to pipeline YAMLs within the `_ingredients` directory. Rather, it is meant to help guide end-users towards a camera-specific implementation of a pipeline rather than a generic one.

Reply:
If these metrics pipelines *are* intended to be directly used by end-users, then we should find another home for them. Either in a camera-specific implementation, or as a camera-agnostic implementation in the top-level `pipelines` directory.

Reply:
Ok, let's leave them where they are.